htdocs/inc.processCheckCardEditRegister.php

```diff
@@ -278,11 +278,12 @@ function fillRfidArrAvailWithUsed($rfidAvailArr, $rfidUsedArr=array()) {
                 // link to existing audiofolder
                 $foldername = $Audio_Folders_Path."/".$post['audiofolder'];
             }
-            $exec = "cd '".$foldername."'; youtube-dl -f bestaudio --extract-audio --audio-format mp3 ".$post['YTstreamURL']." > ".$conf['shared_abs']."/youtube-dl.log; chmod 777 ".$foldername."/* 2>&1 &";
-            exec($exec);
+            $exec = "cd '".$foldername."'; yt-dlp --compat-options youtube-dl -f bestaudio --extract-audio --audio-format mp3 '".$post['YTstreamURL']."' >> ".$conf['shared_abs']."/youtube-download.log 2>&1; chmod 777 ".$foldername."/*;";
```

Comment:
Where is yt-dlp installed? Couldn't find it in packages or requirements.txt

Reply:
It's in the requirements.txt

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/711d6051ab4a24a32e0eadff926dd88d337dc876/requirements.txt#L10

Reply:
I looked in the wrong file 🫣